### ISSUE TITLE
Accordion doesn't have valid accessible structure (WCAG 4.1.2) [QAC]

### ISSUE THREAD

**Issue by ftourangeau** [MeD-DMC/Adapt-accessibilityfixes#212]
**COURSE:**
https://esdc-edsc-adapt.canadacentral.cloudapp.azure.com/preview/5dc446fd27f08836496b6b9e/63a32079b75e454dafed1e1d/
*******************************************************************************
**Page:**
Course Overview page
Lesson 1: Overview of electronic Passport Next Generation
Lesson 2: Domestic Print
Lesson 4: Manual QA

Description:
The accordion widget in the "Key Features of ePNG" section is not properly designed in terms of accessibility. For example:
   • Wrap the accordion buttons in heading elements, e.g. <h4> (heading level should be appropriate for the information architecture of the page)
   • The button element should be the only element inside the heading element. That is, if there are other visually persistent elements, they are not included inside the heading element

For example:
`<h4> `
  `<button>PSS</button>`
`</h4>`
